packages/core-mobile/app/new/routes/(authenticator)/totp/_layout.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Stack } from 'new/components/navigation/Stack'
+import { SignupProvider } from 'new/contexts/SignupProvider'
+
+export default function TotpLayout(): JSX.Element {
+  return (
+    <SignupProvider>
+      <Stack>
+        <Stack.Screen name="authenticatorSetup" />
+        <Stack.Screen name="scanQrCode" />
+        <Stack.Screen name="copyCode" />
+        <Stack.Screen name="verifyCode" />
+      </Stack>
+    </SignupProvider>
```

Comment:
Although it works well as it is now, I think `TotpLayout` should be declared in a place like the root `_layout.tsx`. Also, `signup.tsx` and `totp/_layout.tsx` should be wrapped with a single `<SignupProvider />`.

Reply:
I was actually trying to wrap `signup.tsx` and other totp screens in single `<SignupProvider />`.  But didn't work out well.  I will move totp to root, to use single  `<SignupProvider />` at root.

Reply:
moved to `totp` to root.